models/Campground.js

```diff
@@ -53,11 +53,30 @@ CampgroundSchema.pre(
   }
 );
 
-// Reverse populate with virtuals
+//Cascade delete announcements when a campground is deleted
```

Comment:
(decided with pull requester) cascade delete is not a good choice as the record might be of use later on